sdk/tools/log2lines/log2lines.c

```diff
@@ -633,19 +633,23 @@ main(int argc, const char **argv)
         int i = 1;
         const char *exefile = NULL;
         const char *offset = NULL;
-        char Line[LINESIZE + 1];
+        char LineOut[LINESIZE + 1];
```

Comment:
Can you just stick with your change and don't rename vars in the middle of the process?

Reply:
Rename reverted.